libs/core-ui/src/lib/Cohort/ManualCohortManagement/CohortEditorFilter.tsx

```diff
@@ -228,6 +232,17 @@ export class CohortEditorFilter extends React.Component<ICohortEditorFilterProps
                       this.props.setNumericValue(0, selectedMeta, 1, value);
                     }}
                   />
+                  {this.props.showInvalidMinMaxValueError &&
+                    selectedMeta.featureRange && (
+                      <p className={styles.invalidValueError}>
+                        {localization.formatString(
+                          localization.Interpret.CohortEditor
```

Comment:
I think this should be Core instead of Interpret now:

```
localization.Interpret.CohortEditor
```
should be:

```
localization.Core.CohortEditor
```

it looks like this wasn't updated properly when it was moved to core.

Reply:
I guess, if you prefer, you can create an issue for this... but someone needs to fix it at some point

Reply:
Looks like there are a few changes needed. Created an issue for tracking purpose: https://github.com/microsoft/responsible-ai-toolbox/issues/1340